docking/dock_and_equilibrate.py

```diff
@@ -181,21 +181,23 @@ def dock_and_equilibrate(
                 guest_name,
                 outdir,
                 str(step).zfill(len(str(insertion_steps))),
-                f"ins",
+                "ins",
             )
 
         if report.too_much_force(ctxt, lamb, host_box, combined_bps, u_impls):
             print("Not calculating work (too much force)")
             calc_work = False
-            break
+            continue
```

Comment:
Just checking that you intended to continue as opposed to break

Reply:
yep, that was the bug! there's one spot where it should break and a different spot where it should continue. The bug was that if one compound in an sdf failed, the rest of the compounds in the set wouldn't run at all